src/EventListener/CreateSchemaListener.php

```diff
@@ -46,6 +49,7 @@ public function __construct(AuditManager $auditManager)
 
     /**
      * @return string[]
+     * NEXT_MAJOR: remove this method.
```

Comment:
```suggestion
     * NEXT_MAJOR: remove this method.
     *
     * @return string[]
```
instead ; same for others